### ISSUE TITLE
Remove health drain in mania

### ISSUE THREAD

**Issue by CrazinessUwU** [ppy/osu#8075]
Let me rephrase what im trying to said

This type of Osu Health Bar can't be placed in Osu!Mania becasuse:
 - Osu!Mania have a health bar that didn't drop health in time
 - Osu! health bar have health drop in time although player don't miss anything, and using that type of healthbar for Osu!Mania is IMPOSSIBLE to beat low BPM level ( newbies level)

So this is not Drop to much health, this need to be replace with an Osu!mania health bar 
Fix it pleassee

![image](https://user-images.githubusercontent.com/61653670/75628005-b6d28d80-5c07-11ea-9f6e-617576609917.png)

**Comment by bdach** [comment]
You could have clarified that in the previous one, I'd rename it better and reopen.

**Comment by CrazinessUwU** [comment]
ok thx

**Comment by peppy** [comment]
for now we are aiming to have as much shared across all rulesets. lazer is not aiming to play 1:1 accuracy with stable.

hp drain will not be going anywhere. it may be adjusted but you will need to provide examples of what you think is too harsh rather than a general “i don’t like this”.

**Comment by bdach** [comment]
Seeing that drain is staying I'll close this again. Discussion about its harshness can be continued in #7975.

**Comment by enoslayd** [comment]
There are some Beatmaps that doesnt have notes for like 2 secs and doesnt use a break and that might lead to failing a map especially newbies

**Comment by smoogipoo** [comment]
Yes that’s how HP drain works. The one thing that is likely to change is the scaling factor, since mania beatmaps generally have higher HP drain values due to the osu!stable implementation.

**Comment by mcendu** [comment]
Duplicate of #7370

**Comment by Kosyne** [comment]
@peppy @smoogipoo 
> 
> for now we are aiming to have as much shared across all rulesets. lazer is not aiming to play 1:1 accuracy with stable.

And neither should osumania look to play 1:1 with osu. An alternate gamemode's whole purpose is to play the game differently. Lazer is a **new** project, but it has an **existing** playerbase, so while change isn't bad, there's a bit to consider here.

> hp drain will not be going anywhere. it may be adjusted but you will need to provide examples of what you think is too harsh rather than a general “i don’t like this”.

It's not a case of being harsh, it's simply just an unwelcome change IMO. Looking through the related issues it seems like this feature was added randomly on a whim, to the point another contributor thought it was a bug. That it could be mistaken as a bug should be seen as an indicator of how this change may be received.


That said, to say more than "I don't like this":

* Many existing maps just weren't made with life drain in mind. If this feature is kept without any chance of removal, then this point at least **will HAVE to be addressed**.
* Your application has a legacy of existing content and players that this change will impact.
* A change like this fosters a divide in more hardcore players and more casual players. See the threads full of differing opinions where the lifedrain in the standard mode is discussed, for instance.
* Osumania draws in stepmania players. Now I know very well the intent isn't to be 1:1 with stepmania, but that said, this will almost certainly be something unwelcome for players of _this mode's namesake_.
* Life drain is already a feature with a fair share of, well, strong opinions towards it, so adding it into a mode that didn't have it before seems a little tone deaf.

 I get you guys are trying something new, and I honestly have nothing against that at all! However, I believe that making a change on a whim and taking an attitude of "deal with it" on all the tickets related to this isn't exactly the right approach. Sometimes if it ain't broke, it doesn't need to be fixed, and I think this just might be one of those times.

**Comment by peppy** [comment]
Thanks for your opinion. Please see the disclaimer on the readme of this project. We are not looking at game balance or mechanics in cases where there is subjective opinion involved just yet.

We are not trying "something new". We are unifying implementation to make it easier to develop. If we decide per-implementation nuances are required, they will be added at a later stage.

**Comment by Kosyne** [comment]
> We are not looking at game balance or mechanics in cases where there is subjective opinion involved just yet.

The message I replied to asked for examples, so I obliged.

Anyway, I can understand wanting to make development easier, I work in that field myself, but I do hope this topic is revisited and given more thought and discussion when you guys do reach that phase of the project. Thanks for your time and hope future work goes well.
